docs/lifecycle-hooks.md

```diff
@@ -311,42 +311,42 @@ trait HasPostForm
 
     public $content = '';
 
-    public function hasPostFormMount()
+    public function mountHasPostForm()
     {
         // ...
     }
 
-    public function hasPostFormHydrate()
+    public function hydrateHasPostForm()
     {
         // ...
     }
 
-    public function hasPostFormBoot()
+    public function bootHasPostForm()
     {
         // ...
     }
 
-    public function hasPostFormUpdating()
+    public function updatingHasPostForm()
     {
         // ...
     }
 
-    public function hasPostFormUpdated()
+    public function updatedHasPostForm()
     {
         // ...
     }
 
-    public function hasPostFormRendering()
+    public function renderingHasPostForm()
     {
         // ...
     }
 
-    public function hasPostFormRendered()
+    public function renderedHasPostForm()
     {
         // ...
     }
 
-    public function hasPostFormDehydrated()
+    public function dehydratedHasPostForm()
```

Comment:
Should be `dehydrateHasPostForm`, as the hook method is `dehydrate()`.
See https://github.com/livewire/livewire/blob/ad33d14f9ca37a74f270b2495ec8f99aa3e29ba0/src/Features/SupportLifecycleHooks/SupportLifecycleHooks.php#L112C37-L112C37